mis_builder/report/mis_report_instance_qweb.xml

```diff
@@ -97,12 +98,30 @@
                                                 <t
                                                     t-out="cell and cell.val_rendered or ''"
                                                 />
+                                                <span class="footnote-mis-builder">
+                                                    <t
+                                                        t-out="notes.get(cell.cell_id,{}).get('sequence','')"
```

Comment:
I got a `AttributeError: 'NoneType' object has no attribute 'cell_id'` when generating the pdf with the default demo report on runboat, probably here.

Reply:
In deed, I'll check that, thanks!